package.json

```diff
@@ -21,10 +21,10 @@
     "lint": "yarn eslint --max-warnings=0 && yarn prettier && yarn stylelint",
     "lint:fix": "yarn stylelint --fix && yarn eslint --max-warnings=0 --fix && yarn prettier --write",
     "clean:storybook": "rimraf 'storybook/public'",
-    "clean": "lerna run clean && yarn clean:storybook",
+    "clean": "yarn clean:storybook",
```

Comment:
We should actually be looking through all the repos and cleaning out their dist files with this command 😬

Reply:
Ah right, Lerna did that didn't it... But I can only think of introducing a task runner nx/turbo to execute the same thing